snuba/api.py

```diff
@@ -267,6 +262,12 @@ def parse_and_run_query(validated_body, timer):
         ], key=lambda priority_and_col: priority_and_col[0])
         if prewhere_candidates:
             prewhere_conditions = [cond for _, cond in prewhere_candidates][:settings.MAX_PREWHERE_CONDITIONS]
+            where_conditions = list(filter(lambda cond: cond not in prewhere_conditions, where_conditions))
+
+    where_clause = ''
+    if where_conditions:
+        where_conditions = util.tuplify(where_conditions)
```

Comment:
I don't think there's anything in the preceding block that expects `where_conditions` and it's recursive children to be a tuple and not a list, but I'm not sure and I don't think there is any reasonable way to make sure this is accurate, other than leaning on the tests (and trying to increase type coverage to avoid this kind of thing in the future.) Just watch this carefully to see if anything seems weird during deploy.

Reply:
Cool, thanks for the heads up